acceptance-tests/src/test/java/tech/pegasys/web3signer/dsl/utils/MetadataFileHelpers.java

```diff
@@ -131,6 +133,8 @@ public void createHashicorpYamlFileAt(
       signingMetadata.put("token", node.getVaultToken());
       signingMetadata.put("keyType", node.getKeyType().toString());
 
+      httpProtocolVersion.ifPresent(version -> signingMetadata.put("httpProtocolVersion", version));
```

Comment:
nit: "magic string" - is there not a type we can use here? e.g. `HttpClient.Version`?

Reply:
this is part of our Acceptance Test DSL class. All the keys and values are string literals.